pkg/terraform/install.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 
 	"get.porter.sh/porter/pkg/exec/builder"
+	"github.com/tidwall/gjson"
```

Comment:
I wanted to call out that porter already brings in a dependency on github.com/PaesslerAG/jsonpath which is pretty similar to what you are using gjson for. You may be able to reuse that dependency and not add another.

Reply:
Turns out I was totally wrong about what y'all were doing with that library. Ignore me!